### PR TITLE
React UI: Improve styling of autocomplete sections

### PR DESCRIPTION
**Pull request by juliusv**
I removed the Card-related components and went back to normal `<ul>`/`<li>`,
since the style that Cards added just got in the way (like adding extra
borders and rounding, etc.), and from the examples at
https://getbootstrap.com/docs/4.3/components/card/, it doesn't seem like
multiple Cards are meant to be used as part of a larger list
(style-wise).

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@boyskila Screenshot:

autocomplete

**Comment by boyskila** [comment]
> I removed the Card-related components and went back to normal `<ul>`/`<li>`,
> since the style that Cards added just got in the way (like adding extra
> borders and rounding, etc.), and from the examples at
> https://getbootstrap.com/docs/4.3/components/card/, it doesn't seem like
> multiple Cards are meant to be used as part of a larger list
> (style-wise).
> 
> Signed-off-by: Julius Volz [julius.volz@gmail.com](mailto:julius.volz@gmail.com)

Actually only two Cards are rendered in this case - per section, but yes Cards has his own styles and make no sense to be used if you want to achieve something different :)

**Comment by juliusv** [comment]
@boyskila updated this ontop of current master and fixed test. What do you think?

File: web/ui/react-app/src/ExpressionInput.tsx [hunk 1]
```diff
@@ -107,12 +97,12 @@ class ExpressionInput extends Component<ExpressionInputProps, ExpressionInputSta
                           },
                         });
                         return (
-                          <SanitizeHTML tag={ListGroupItem} {...itemProps} allowedTags={['strong']}>
+                          <SanitizeHTML tag={'li'} {...itemProps} allowedTags={['strong']}>
```
Comment (boyskila) [inline web/ui/react-app/src/ExpressionInput.tsx]: {'li'} -> "li"

Comment (juliusv) [reply web/ui/react-app/src/ExpressionInput.tsx]: Done.

File: web/ui/react-app/src/App.css [hunk 2]
```diff
@@ -85,9 +90,11 @@ button.execute-btn {
...
-    background: rgba(240, 230, 140, 0.4);
-    height: 30px;
+.autosuggest-dropdown-list li.autosuggest-dropdown-header {
+    margin-bottom: 5px;
+    background-color: #bfdeff;
+    height: 25px;
+    font-size: 10px;
     display: flex;
     align-items: center;
     text-transform: uppercase;
```
[the remark this answers is not in the document]
Comment (juliusv) [reply web/ui/react-app/src/App.css]: Thanks, I removed the rounded corners and some of the other stuff, but added a `margin-top: -6px` to bring the dropdown to the correct top position again.

**Comment by juliusv** [comment]
It now looks like this:

auto

**Comment by juliusv** [comment]
(I also centered the section header texts and changed the spacing a bit)

File: web/ui/react-app/src/App.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (boyskila) [reply web/ui/react-app/src/App.css]: Looks good

[the remark this answers is not in the document]
Comment (juliusv) [reply web/ui/react-app/src/App.css]: Is that a :+1: for the PR? :)

[the remark this answers is not in the document]
Comment (boyskila) [reply web/ui/react-app/src/App.css]: +1

**Pull request merged**